Constitution.md

```diff
@@ -207,6 +225,42 @@ If there are no written minutes, written notice, and/or the information presente
 
 If an executive officer is dismissed, the club must follow [**Section 3: Club Changeover**](#section-3-club-changeover) in order to follow the proper procedure(s) outlined to fill the vacant position(s).
 
+## Article 2d: Committees, Councillors, and Community Advocates
+
+### Committees
+A Committee is a group of council members that forms to complete a well-defined goal. Any Executive Officer may propose a committee by outlining its purpose and the criteria of its members (if any). The Council will vote on the formation of this Committee, and upon its formation, the proposing Executive Officer will lead it.
+
+A Committee leader will have the following duties:
+1. Appoint Committee members from Council.
+2. If applicable, announce the formation of the Committee to Members of the Association so that they may apply for it.
+3. Outline the necessary tasks for the Committee to complete its goal.
+4. Assign tasks to Committee members.
+5. Call Committee meetings.
+6. Report on Committee activites and progress at Council meetings.
+7. Appoint a proxy Committee leader in the event that they become unavailable to lead.
+
+If a Committee leader deems the Committee's goal to be fulfilled, they may propose its dissolution to be voted on by the Council.
+
+### Councillors
+Councillors are non-voting members of council. Members of CSSA may apply for a Councillor position through an application that is publicly available throughout the year. Councillor applicants may indicate their interest to join an active committee. Once appointed by the Council, a Councillor may assume a general purpose volunteer role, or be appointed to a committee that they express interest in.
+
+### Community Advocates
+Applicants will have the option to self-nominate for a Community Advocate position on their Councillor application. However, self-nominations for Community Advocates will close [WITHIN SOME TIME LIMIT], after which the option will be removed from application form.
+
+Applicants may choose one or more of the following Community Advocate roles:
+- Women's Advocate
+- Accessibility Advocate
+- 2SLGBTQIA+ Students' Advocate
+- Indigenous Students' Advocate
+- Black Students' Advocate
+- International Students' Advocate
+- Mature Students' Advocate
+- Racialized Students' Advocate
+
+An applicant must self-identify as a member of the community they are self-nominating to advocate for.
+
+If multiple applicants self-nominate for the same Community Advocate role, the role will be voted on by CSSA members that self-identify as members of that community. The Director of Advocacy is responsible for overseeing and administering this process.
```

Comment:
what if we dont have members who self-identify in that community?

Reply:
It's ok to have a vacant role. Even the umsu mature rep role has been vacant for a few years. Giving the option from someone of the community to represent would be ideal but if not the other advocates would likely still be working together to make sure everyone is included

Reply:
Agree w/Isham, I'll put that in more explicitly ty!